utils/oscap-ssh

```diff
@@ -238,6 +243,9 @@ fi
 if [ "$TARGET_SYSCHAR" != "" ]; then
     scp -o ControlPath="$MASTER_SOCKET" -P "$SSH_PORT" "$SSH_HOST:$REMOTE_TEMP_DIR/syschar.xml" "$TARGET_SYSCHAR" || die "Failed to copy the OVAL syschar file back to local machine!"
 fi
+if [ "$OVAL_RESULTS" == "yes" ]; then
+    scp -o ControlPath="$MASTER_SOCKET" -P "$SSH_PORT" "$SSH_HOST:$REMOTE_TEMP_DIR/*-oval.xml.result.xml" "./" || die "Failed to copy OVAL result files back to local machine!"
```

Comment:
oval file names don't necessarily have to end with `-oval.xml`, I suggest to instead use `*.result.xml`. Technically it doesn't even have to have the xml extension.

Reply:
I started with just result.xml, it felt too vague, so I added more - you are right though. SSG naming convention is not something to hardcode in oscap.